axelrod/strategies/axelrod_first.py

```diff
@@ -523,9 +695,9 @@ class Tullock(Player):
     - Tullock: [Axelrod1980]_
     """
 
-    name = "Tullock"
+    name = "First tournament by Tullock"
     classifier = {
-        "memory_depth": 11,  # long memory, modified by init
+        "memory_depth": 11,
```

Comment:
I think this one might be infinite actually since it needs to know if it's within the first 11 rounds or not

Reply:
Yeah good call!